addons/event_sale/models/event.py

```diff
@@ -13,7 +13,6 @@ class event_event(models.Model):
         'event.event.ticket', 'event_id', string='Event Ticket',
         default=lambda rec: rec._default_tickets(), copy=True)
 
-    badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
```

Comment:
Reason for moving the field 'badge_back' into event is to make the right side badge of report to make editable when only event installed.